frontend/package.json

```diff
@@ -47,7 +47,8 @@
     "yarn-run-all": "^3.1.1"
   },
   "dependencies": {
-    "eslint": "^8.0.1"
+    "eslint": "^8.0.1",
+    "moment": "^2.29.4"
```

Comment:
this is not the right place to install that package

Reply:
Moved to zc_messaging